CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Enh #312: Change property `Schema::$typeMap` to constant `Schema::TYPE_MAP` (@Tigrov)
 - Bug #314: Fix `Command::insertWithReturningPks()` method for empty values (@Tigrov)
+- Enh #319: Resolve psalm issues (@Tigrov)
```

Comment:
```suggestion
- Enh #319: Minor refactoring of `DDLQueryBuilder` and `Schema` (@Tigrov)
```

"Resolve psalm issues" is not about public interface of package and too common description of changes. Suggest replace it to refactoring...

Reply:
Moved to another PR #320